earth_enterprise/src/scons/khEnvironment.py

```diff
@@ -159,6 +161,192 @@ def EmitBuildDateStrfunc(target, build_date):
   return 'EmitBuildDate(%s, %s)' % (target, build_date)
 
 
+def EmitVersionHeaderFunc(target, backupFile):
+  """Emit version information to the target file."""
+
+  versionStr = GetVersion(backupFile)
+  longVersionStr = GetLongVersion(backupFile)
+
+  fp = open(target, 'w')
```

Comment:
Minor: You can use a `with`statement for writing to files.  It handles exceptions, and is shorter.